example/python/irohalib.md

```diff
@@ -0,0 +1,30 @@
+# Python Iroha Lib Description
+
+irohalib.py is a possible implemetation of a platform-independent way of interaction with Iroha network via Python 3.
```

Comment:
possible -> example?

Reply:
Possible is ok here and perfectly reflects the initial intention.

Thanks for commenting anyway - I have found a type in "implementation" word!